packages/gert/install

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+set -e
+
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0xb3cf35c315b55a9f
```

Comment:
Let's add an OS version check for below bionic or focal, whichever has a new enough libgit2 to make gert happy.

Reply:
ah yes